crypto/relic_build.sh

```diff
@@ -42,7 +42,8 @@ if [[ "$ARCH" =~ ^(arm64|armv7|armv7s)$ && "${CC_VERSION_STR[0]}" =~ (clang)  ]]
     #  the "-march=native" option is not supported with clang on ARM
     MARCH=""
 else
-    MARCH="-march=native"
+    # Target standard x86-64 CPU. Avoids AVX512 SIGILL: illegal instruction
+    MARCH="-march=x86-64"
```

Comment:
@yagop - Something that came up from the discussion we had about using a non-AVX512 compile- 

> The problem with non-AVX512 is we leave perf on the table for everyone.
> One thing that would be useful is if netgo & Kraken could give us a /proc/cpuinfo, or, alternatively, the very precise spec of the CPUs they use.
> It would let us know if we can seek perf elsewhere (e.g. ADX)